logging/main.js

```diff
@@ -0,0 +1,35 @@
+var Trace;
```

Comment:
The name "Trace" is a bit of an artifact of my original naming conventions. I recognize now it logs errors and other things, but I haven't had the heart to change the name.